src/applications/ivc-champva/10-7959C/config/submitTransformer.js

```diff
@@ -50,6 +50,8 @@ export default function transformForSubmit(formConfig, form) {
   // a status event notification
   copyOfData.primaryContactInfo = getPrimaryContact(copyOfData);
 
+  copyOfData.statementOfTruthSignature = copyOfData.signature;
```

Comment:
Makes sure we have the signature property that the PDF stamper will be looking for.